lib/assembly-objectfile/object_file.rb

```diff
@@ -146,71 +146,13 @@ def mimetype
         check_for_file
         mimetype = ''
         mime_type_order.each do |mime_type_method|
-          mimetype = public_send("#{mime_type_method}_mimetype") if VALID_MIMETYPE_METHODS.include?(mime_type_method)
+          mimetype = send("#{mime_type_method}_mimetype") if VALID_MIMETYPE_METHODS.include?(mime_type_method)
           break if mimetype.present?
         end
         mimetype
       end
     end
 
-    # Returns mimetype information using the manual override mapping (based on a file extension lookup)
-    # @return [String] mime type for supplied file if a mapping exists for the file's extension
-    # @example
-    #   source_file = Assembly::ObjectFile.new('/input/path_to_file.json')
-    #   puts source_file.override_mimetype # 'application/json'
-    def override_mimetype
```

Comment:
for all of the methods made private, I carefully checked that there were no apps using this gem that called these methods, and/or did a github search on the method name in org:sul-dlss.